streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredSessionStoreTest.java

```diff
@@ -133,45 +127,34 @@ public void before() {
     }
 
     private void init() {
-        replay(innerStore, context);
         store.init((StateStoreContext) context, store);
     }
 
     @SuppressWarnings("deprecation")
     @Test
     public void shouldDelegateDeprecatedInit() {
-        final SessionStore<Bytes, byte[]> inner = mock(SessionStore.class);
         final MeteredSessionStore<String, String> outer = new MeteredSessionStore<>(
-            inner,
+            innerStore,
             STORE_TYPE,
             Serdes.String(),
             Serdes.String(),
             new MockTime()
         );
-        expect(inner.name()).andStubReturn("store");
```

Comment:
This is unneeded here and elsewhere due to being initialised in the setup method.